File: Assets/Scripts/CSharp/Movables.cs
using UnityEngine;
using System.Collections;

public class Movables : MonoBehaviour 
{
	public float additive = 0.0f;
	
	void Start()
	{
		Vector3 tmpPos;
		
		if((Screen.width*1.0)/(Screen.height*1.0)<1.7)
		{
			tmpPos = transform.position;
			tmpPos.x -= 9 + additive; 
			transform.position = tmpPos;
		}
		if((Screen.width*1.0)/(Screen.height*1.0)<1.5)
		{
			tmpPos = transform.position;
			tmpPos.x -= 8 + additive;
			transform.position = tmpPos;
		}
	}
	
	void Update () 
	{
	
	}
}

File: Assets/Scripts/Movables.js
#pragma strict
var additive:float = 0.0;
function Start()
{
	if((Screen.width*1.0)/(Screen.height*1.0)<1.7)
	{
		transform.position.x -= 9 + additive;
	}
	if((Screen.width*1.0)/(Screen.height*1.0)<1.5)
	{
		transform.position.x -= 8 + additive;
	}
}

function Update () {

}